### DISCUSSION TITLE
Welcome to Eaglescope Discussions!

### DISCUSSION THREAD

**Discussion by birm** [sharmalab/eaglescope#95] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bemijonathan** [comment]
Hello guys my name is jonathan, i am interested in working on this project as a part of GSOC 2024,  i have a  background in cell biology and human anatomy, i am currently studying my masters in artificial intelligence, i am more than excited to contribute to this project. 

Please i would love to know what email we would submit our draft proposals
